manual_hook.rb

```diff
@@ -5,10 +5,8 @@
 def setup_dns(domain, txt_challenge)
   dns = Resolv::DNS.new;
   acme_domain = "_acme-challenge."+domain; 
-  puts "Create TXT record for the domain: \"#{acme_domain}\". TXT record:"
+  puts "Checking TXT record for the domain: \"#{acme_domain}\". TXT record:"
   puts "\"#{txt_challenge}\""
-  puts "Press enter when DNS has been updated..."
-  $stdin.readline()
   resolved = false;
```

Comment:
> `resolved = false;`

i'd move this below after line 7, since this is setting up a normal variable (my bad, not yours)